features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/Throttling/Subscription/AddEdit.jsx

```diff
@@ -237,7 +237,7 @@ function AddEdit(props) {
                 setValidRoles(result.body.permissions
                     && result.body.permissions.roles
                     ? result.body.permissions.roles
-                    : []);
+                    : ['Internal/everyone']);
```

Comment:
Is it really needed to add it? we can have subscription policies without Internal/everyone permission..

Reply:
In the REST API definition, the parameter `roles` is marked as required one - [1]. We should either send a default value or mark it as a not required parameter.In earlier versions, also we are sending this default value. 
Then shall we mark it as a not required parameter. WDYT?

[1] https://github.com/wso2/carbon-apimgt/blob/master/components/apimgt/org.wso2.carbon.apimgt.rest.api.admin.v1/src/main/resources/admin-api.yaml#L3256

Reply:
I think we can remove it as a required one.. 
@chamilaa did a similar fix recently..

Reply:
+1. Yes, we can send null if permission is not required for a policy. Handled it in UI by adding a new option `None`:

![image](https://user-images.githubusercontent.com/19728269/114391191-ce26cd00-9bb4-11eb-922b-67094cc63c16.png)

Reply:
Fixed

Reply:
Great!